scripts/attractions/AttractionPreview.js

```diff
@@ -0,0 +1,40 @@
+import { useOddities } from "./AttractionProvider.js"
```

Comment:
Can we change the name of all the files that deal with oddities from Attractions to Oddities?